lib/concurrent-ruby/concurrent/collection/ruby_timeout_queue.rb

```diff
@@ -0,0 +1,79 @@
+module Concurrent
+  module Collection
+    # @!visibility private
+    # @!macro ruby_timeout_queue
+    class RubyTimeoutQueue < ::Queue
```

Comment:
Let's use `private_constant` on this to make it very clear

Reply:
Done.